test/functional/pdo_sqlsrv/pdo_prepare_emulatePrepare_unicode.phpt

```diff
@@ -70,9 +70,21 @@ try {
     print_r("Prepare with emulate prepare and and SQLSRV_ENCODING_SYSTEM:\n");
     $stmt = prepareStmt($conn, $query, $options, PDO::PARAM_STR, 0, PDO::SQLSRV_ENCODING_SYSTEM);
     $row = $stmt->fetch(PDO::FETCH_ASSOC);
-    print_r($row);
-    if ($stmt->rowCount() == 0) {
-        print_r("No results for this query\n");
+    // Linux supports binding UTF8 data with SQLSRV_ENCODING_SYSTEM since the default encoding in Linux is UTF8
+    $success = true;
+    if (!(strtoupper( substr( php_uname( 's' ),0,3 ) ) === 'WIN') && isAEConnected()) {
+        if ($row['name'] != "가각" || $row['status'] != 1 || $row['age'] != 30) {
```

Comment:
Not just Linux but also Mac? Perhaps also explains why AE is related to this condition?
in the else block please annotate that in Windows the default is not utf-8